Formula/dcos-cli.rb

```diff
@@ -0,0 +1,20 @@
+class DcosCli < Formula
+  desc "The command-line for your datacenter!"
```

Comment:
`"Cross-platform command line utility to manage DC/OS clusters"` would be more helpful.